ports/stm32/mboot/mboot_pack_dfu.py

```diff
@@ -228,11 +230,22 @@ def verify_pack_dfu(keys, filename):
                 chunk, 0, MBOOT_PACK_HYDRO_CONTEXT, keys.secretbox
             )
             assert chunk is not None
+            binary_data += chunk
             if header[1] == MBOOT_PACK_CHUNK_FW_GZIP:
                 chunk = zlib.decompress(chunk, wbits=-15)
             full_sig.update(chunk)
     full_sig_pass = full_sig.final_verify(actual_sig, keys.sign_pk)
     assert full_sig_pass
+    return binary_data
+
+
+def unpack_dfu(keys, args):
+    # Load previously generated keys.
+    keys.load()
+
+    # build a DFU file from the decrypted binary data
+    binary_data = verify_pack_dfu(keys, args.infile[0])
+    dfu.build(args.outfile[0], [[{"address": 0x08008000, "data": binary_data}]])
```

Comment:
The 0x08008000 cannot be hard coded.  `verify_pack_dfu` must return this value.

Reply:
Now returned from `verify_pack_dfu`